tests/test_tallies/test_tallies.py

```diff
@@ -171,32 +172,18 @@ def _build_inputs(self):
         all_nuclide_tallies[0].estimator = 'collision'
 
         self._input_set.tallies = Tallies()
-        self._input_set.tallies.add_tally(azimuthal_tally1)
-        self._input_set.tallies.add_tally(azimuthal_tally2)
-        self._input_set.tallies.add_tally(azimuthal_tally3)
-        self._input_set.tallies.add_tally(azimuthal_tally4)
-        self._input_set.tallies.add_tally(cellborn_tally)
-        self._input_set.tallies.add_tally(dg_tally)
-        self._input_set.tallies.add_tally(energy_tally)
-        self._input_set.tallies.add_tally(energyout_tally)
-        self._input_set.tallies.add_tally(transfer_tally)
-        self._input_set.tallies.add_tally(material_tally)
-        self._input_set.tallies.add_tally(mu_tally1)
-        self._input_set.tallies.add_tally(mu_tally2)
-        self._input_set.tallies.add_tally(mu_tally3)
-        self._input_set.tallies.add_tally(polar_tally1)
-        self._input_set.tallies.add_tally(polar_tally2)
-        self._input_set.tallies.add_tally(polar_tally3)
-        self._input_set.tallies.add_tally(polar_tally4)
-        self._input_set.tallies.add_tally(universe_tally)
-        [self._input_set.tallies.add_tally(t) for t in score_tallies]
-        [self._input_set.tallies.add_tally(t) for t in flux_tallies]
-        self._input_set.tallies.add_tally(scatter_tally1)
-        self._input_set.tallies.add_tally(scatter_tally2)
-        [self._input_set.tallies.add_tally(t) for t in total_tallies]
-        self._input_set.tallies.add_tally(questionable_tally)
-        [self._input_set.tallies.add_tally(t) for t in all_nuclide_tallies]
-        self._input_set.tallies.add_mesh(mesh_2x2)
+        self._input_set.tallies += (
```

Comment:
Glad to see that the `CheckedList` nature of `Tallies` is fully tested here now.